src/guc.c

```diff
@@ -1105,6 +1121,20 @@ _guc_init(void)
 							 /* show_hook= */ NULL);
 #endif
 
+	DefineCustomEnumVariable(MAKE_EXTOPTION("default_hypercore_use_access_method"),
+							 gettext_noop(
+								 "Whether to use access method for compression by default."),
+							 gettext_noop(
+								 "Sets the global default for compression technique to use."),
```

Comment:
Misleading since it is not about compression technique (sounds like it is about type of compression used).

```suggestion
								 "Sets the global default for using Hypercore TAM when compressing chunks."),
```